packages/grid/_modules_/grid/models/api/i18nApi.ts

```diff
@@ -0,0 +1,17 @@
+import { LocaleText } from '../gridOptions';
+
+export type LocaleTextValue = string | React.ReactNode | Function;
+
+export type TranslationKeys = keyof LocaleText;
+
+/**
+ * The i18n API interface that is available in the grid [[apiRef]].
```

Comment:
```suggestion
 * The grid i18n API [[apiRef]].
```

What does [[apiRef]] represent?